sources/packages/backend/libs/test-utils/src/factories/cas-supplier.ts

```diff
@@ -27,7 +27,8 @@ export async function saveFakeCASSupplier(
     student?: Student;
   },
   options?: {
-    supplierStatus: SupplierStatus;
+    supplierStatus?: SupplierStatus;
+    isValid?: boolean;
```

Comment:
When the values are related to the target entity of the factory we use the initial values. Please check similar factories.